rpc/src/rpc.rs

```diff
@@ -507,6 +537,53 @@ impl JsonRpcRequestProcessor {
         })
     }
 
+    pub fn get_secondary_index_key_size(
+        &self,
+        index_key: &Pubkey,
+        config: Option<RpcSecondaryIndexConfig>,
+    ) -> Result<RpcResponse<HashMap<RpcAccountIndex, usize>>> {
+        // Acquire the bank
+        let bank = if config.is_some() {
+            self.bank(config.unwrap().commitment)
+        }
+        else{ self.bank(None) };
```

Comment:
This suggestion will need adjusting depending on what config/commitment you land on, but you can construct this much more nicely:
```suggestion
        let bank = self.get_bank_with_config(config.unwrap_or_default())?;
```